.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Could we actually remove these .idea files from the PR? Maybe add them to the `.gitignore`